components/WhatsNewSection.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <section id="whats-new">
+    <v-container>
+      <v-stack
+        align="start"
+        direction="column"
+        sm-direction="column"
+        gap="1.875rem"
+        sm-gap="1.25rem"
+      >
+        <v-heading
+          class="features-heading"
+          :level="2"
+          line-height="1.2"
+          :weight="700"
+        >
+          What's New
+        </v-heading>
+        <v-text
+          class="features-description"
+          color="secondary"
+          :weight="400"
+          line-height="1.4"
+        >
+          Check out release notes here for all the new updates since previous
+          testnet.
+        </v-text>
+      </v-stack>
+      <v-stack
+        justify="space-between"
+        :grow="1"
+        sm-direction="column"
+        class="features-list"
+      >
+        <v-blurb
```

Comment:
In the mobile design, the features icons are aligned to the title text. 

![Screenshot 2022-05-30 at 10 30 18 PM](https://user-images.githubusercontent.com/11095038/171035574-745ba294-e70b-4459-8797-c0d38dd20e6a.png)